vendor/assets/stylesheets/normalize.css

```diff
@@ -0,0 +1,419 @@
+/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
i like idea of installing via npm but i think the added complexity of then copying files into ./vendor or telling rails to look in ./node_modules for source files isn't worth it right now -- i think we should invest in figuring that out while still using the rails asset pipeline the next time we want to incorporate a third-party css/js file